### PR TITLE
Tests/603

### PR DESCRIPTION
**Pull request by IlyaBelik**
Wrote Unit tests for Size Redux: 
-actions,
-reducer,
-saga

image

image

image

### CODE REVIEW

File: src/redux/sizes/tests/sizes.actions.test.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      type: REMOVE_SIZE_FROM_STATE,
+      payload: mockSize
+    });
+  });
+
+  it('should update size', () => {
+    const filteredSize = mockSizes.items.filter((item) => item._id === mockId);
+    const updatedSize = {
+      ...filteredSize,
+      name: [
```
Comment (Haytham1337) [inline src/redux/sizes/tests/sizes.actions.test.js:91]: remove this array with magic strings please, you can get tham from variables file

File: src/redux/sizes/tests/sizes.sagas.test.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      .hasFinalState({
+        Size: {
+          ...mockSizesState,
+          sizesLoading: true
+        }
+      })
+      .run()
+      .then((result) => {
+        const { allEffects: analysis } = result;
+        const analysisPut = analysis.filter((e) => e.type === 'PUT');
```
Comment (Haytham1337) [inline src/redux/sizes/tests/sizes.sagas.test.js:178]: remove magic strings please and use constants

File: src/redux/sizes/tests/sizes.sagas.test.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        const analysisCall = analysis.filter((e) => e.type === 'CALL');
```
Comment (Haytham1337) [inline src/redux/sizes/tests/sizes.sagas.test.js:179]: same

File: src/redux/sizes/tests/sizes.sagas.test.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+          ...mockSizesState,
+          sizeLoading: true,
+          sizeError: { e: mockError }
+        }
+      })
+      .run()
+      .then((result) => {
+        const { allEffects: analysis } = result;
+        const analysisPut = analysis.filter((e) => e.type === 'PUT');
+        const analysisCall = analysis.filter((e) => e.type === 'CALL');
```
Comment (Haytham1337) [inline src/redux/sizes/tests/sizes.sagas.test.js:207]: same

File: src/redux/sizes/tests/sizes.actions.test.js (hunk 1 above)
Comment (IlyaBelik) [reply src/redux/sizes/tests/sizes.actions.test.js]: done

**Review by AndriyLymych — APPROVED** [review]

**Review by Haytham1337 — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Review by grebiniuk — APPROVED** [review]
All good, nice job

**Pull request merged**